packages/oauth/src/providers/linkedin.ts

```diff
@@ -104,24 +96,26 @@ export const linkedin = <_Auth extends Auth>(auth: _Auth, config: Config) => {
 	} as const satisfies OAuthProvider;
 };
 
+type Locale = {
+	country: string;
+	language: string;
+}
 type LinkedinProfileResponse = {
-	id: string;
-	localizedFirstName: string;
-	localizedLastName: string;
-	profilePicture: {
-		"displayImage~"?: {
-			elements?: Array<{
-				identifiers?: Array<{
-					identifier?: string;
-				}>;
-			}>;
-		};
-	};
+	sub: string;
+	name: string;
+	email: string;
+	email_verified: boolean;
+	family_name: string;
+	family_name: string;
+	locale: Locale,
+	picture: string | null;
 };
 
 export type LinkedinUser = {
 	id: string;
 	firstName: string;
 	lastName: string;
+	email: string;
+	emailVerified: boolean;
 	profilePicture?: string;
```

Comment:
I'm assuming `profilePicture` will always be defined since `LinkedinProfileResponse.picture` is always defined

```suggestion
	profilePicture: string;
```